fronts-client/src/components/card/article/ArticleBody.tsx

```diff
@@ -220,6 +227,38 @@ const articleBodyDefault = React.memo(
 			imageCriteria.heightAspectRatio ===
 				portraitCardImageCriteria.heightAspectRatio;
 
+		const [mainMediaIsSelfHosted, setMainMediaIsSelfHosted] =
```

Comment:
[optional] Typically the `is` part of a boolean goes at the front of the word, to make it easily identifiable as a boolean, i.e. `isMainMediaSelfHosted` instead of `mainMediaIsSelfHosted`